compiler/luci-interpreter/src/core/RuntimeGraph.cpp

```diff
@@ -29,15 +29,22 @@ class RuntimeGraph::TensorAllocPlan
   std::vector<std::vector<Tensor *>> _alloc_plan;
   std::vector<std::vector<Tensor *>> _dealloc_plan;
   bool _valid = false;
+  IMemoryManager *_memory_manager;
 
 public:
+  explicit TensorAllocPlan(IMemoryManager *memory_manager);
   void invalidate() { _valid = false; }
   bool isValid() const { return _valid; }
   void build(const RuntimeGraph &graph);
   void allocate(size_t kernel_index) const;
   void deallocate(size_t kernel_index) const;
 };
 
+RuntimeGraph::TensorAllocPlan::TensorAllocPlan(IMemoryManager *memory_manager)
+  : _memory_manager(memory_manager)
+{
```

Comment:
Note:
In such cases we insert comment, like
```suggestion
{
  // Do nothing
```